ee/allowedcmd/cmd_darwin.go

```diff
@@ -6,148 +6,147 @@ package allowedcmd
 import (
 	"context"
 	"errors"
-	"os/exec"
 )
 
-func Airport(ctx context.Context, arg ...string) (*exec.Cmd, error) {
+func Airport(ctx context.Context, arg ...string) (*TracedCmd, error) {
 	return validatedCommand(ctx, "/System/Library/PrivateFrameworks/Apple80211.framework/Versions/Current/Resources/airport", arg...)
 }
 
-func Bioutil(ctx context.Context, arg ...string) (*exec.Cmd, error) {
+func Bioutil(ctx context.Context, arg ...string) (*TracedCmd, error) {
 	return validatedCommand(ctx, "/usr/bin/bioutil", arg...)
 }
 
-func Bputil(ctx context.Context, arg ...string) (*exec.Cmd, error) {
+func Bputil(ctx context.Context, arg ...string) (*TracedCmd, error) {
 	return validatedCommand(ctx, "/usr/bin/bputil", arg...)
 }
 
-func Brew(ctx context.Context, arg ...string) (*exec.Cmd, error) {
+func Brew(ctx context.Context, arg ...string) (*TracedCmd, error) {
 	for _, p := range []string{"/opt/homebrew/bin/brew", "/usr/local/bin/brew"} {
 		validatedCmd, err := validatedCommand(ctx, p, arg...)
 		if err != nil {
 			continue
 		}
 
-		validatedCmd.Env = append(validatedCmd.Environ(), "HOMEBREW_NO_AUTO_UPDATE=1")
+		validatedCmd.Env = append(validatedCmd.Env, "HOMEBREW_NO_AUTO_UPDATE=1")
```

Comment:
I think this will change the behavior, resulting in a different set of vars in `validatedCmd.Env` than before. (If `cmd.Env` is nil when a command starts running, then the command will automatically add in `os.Environ`. If anything is set in `cmd.Env` prior to command running, it won't add in `os.Environ`.) So before, `validatedCmd.Env` contained both `os.Environ` plus `HOMEBREW_NO_AUTO_UPDATE=1`, and now it will only contain `HOMEBREW_NO_AUTO_UPDATE=1`. I don't know if that matters for this command or not, but maybe it would be safer to add an `Environ()` wrapper command to `TracedCmd` too?